packages/tools/pie-wrapper-react/__tests__/wrapper.test.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs-extra';
+import path from 'path';
+import { addReactWrapper } from '../scripts/add-react-wrapper';
+import removeReactWrapper from '../scripts/remove-react-wrapper';
```

Comment:
Is it possible to make these imports consistent? Looks like you're using a default export for `removeReactWrapper` and a named export for `addReactWrapper`. I don't have a preference either way, but I'd probably make sure that they are importable in the same way for consistency.